.github/workflows/dev_cd.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - "dev"
       - "main"
+      - "cd"
```

Comment:
We generally dont have any branches that fit to this name ("cd"). Why do we need this branch name?